lib/util.js

```diff
@@ -371,10 +371,11 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
+  var symbolKeys = Object.getOwnPropertySymbols(value);
```

Comment:
Yup. That is how `Object.getOwnPropertySymbols` works. It is like the conversation between it and `Object.getOwnPropertyNames` is:

* **`Object.getOwnPropertyNames`**: Hey, Symbol. You may join. But don't expect me to list you, even if you're enumerable...
* **`Symbol`**: :neutral_face:
* **`Object.getOwnPropertySymbols`**: Gee, did you see that guy? Tell you what, Symbol. I'll list all of you, regardless!
* **`Symbol`**: :worried:
* **`Object.getOwnPropertyNames`**: :frowning:
* **`Object.getOwnPropertySymbols`**: :blush: 
* **`Symbol`**: :confused: 

I wouldn't mind changing it to exclude non-enumerable symbol-keyed props. Should I?